tests/test_scripts.py

```diff
@@ -1,4 +1,5 @@
 #
+
```

Comment:
Please always check your commits for extra random lines like this and remove them.